homework-custom-lodash/dropWhile.js

```diff
@@ -1,14 +1,46 @@
-const drop = require("./drop.js");
-//:TODO Доделать метод
+const drop = require("./drop");
+const forEach = require('./helpers/forEach');
+const isArrayLike = require('./helpers/isArrayLike');
+const isObjectLike = require('./helpers/isObjectLike');
+const isPartialEqual = require('./helpers/isPartialEqual');
+
+
 const dropWhile = function dropWhileMethod(array, predicate) {
   let dropNumber = {};
-  let dropArray = {};
-  for (let i = 0; i < array.length; i++) {
-    let element = array[i];
-    if (!predicate(element, i)) {
-      dropNumber = i;
-    }
-    dropArray = drop(array, dropNumber);
+  let dropArray = [];
+  if(typeof predicate === 'function') {
+      forEach(array, (element,index) => {
+          if(!predicate(element,index)) {
+              dropNumber = index;
+          }
+          dropArray = drop(array, dropNumber);
+      })
+  }
+  if(typeof predicate === 'object') {
```

Comment:
`isObjectLike`

Reply:
i resolve conflicts now, one moment

Reply:
i'm so sorry for this, but I did't understand why conflicts formed after my last commits... in my code editor when i am on the master branch, i don’t have a folder at all homework-custom-lodash, but on the homework-lodash branch i have this folder, and i can't understand why conflicts formed...